### PR TITLE
build 02: add machine configs to use crun

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Peter Hunt~ <pehunt@redhat.com>

### CODE REVIEW

**Comment by haircommander** [comment]
cc @jmguzik

**Comment by jmguzik** [comment]
/hold 
/cc @hongkailiu 
@hongkailiu please ask openshift next if we are good to go.

File: clusters/build-clusters/02_cluster/assets/99-crun-control-plane-machineconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    machineconfiguration.openshift.io/role: control-plane
+  name: 99-control-plane-crun
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - contents:
+            source: data:text/plain;charset=utf-8;base64,W2NyaW8ucnVudGltZV0KZGVmYXVsdF9ydW50aW1lID0gImNydW4iCltjcmlvLnJ1bnRpbWUucnVudGltZXMuY3J1bl0K
```
Comment (hongkailiu) [inline clusters/build-clusters/02_cluster/assets/99-crun-control-plane-machineconfig.yaml:14]: Just for the record:
```console
$ echo -n "W2NyaW8ucnVudGltZV0KZGVmYXVsdF9ydW50aW1lID0gImNydW4iCltjcmlvLnJ1bnRpbWUucnVudGltZXMuY3J1bl0K" |base64 -d
[crio.runtime]
default_runtime = "crun"
[crio.runtime.runtimes.crun]
```

Comment (hongkailiu) [reply clusters/build-clusters/02_cluster/assets/99-crun-control-plane-machineconfig.yaml:14]: Oh. Just saw https://github.com/openshift/release/pull/31199#issuecomment-1209782307

**Comment by hongkailiu** [comment]
/lgtm

I will wait a day or two before canceling the hold to let TRT be aware.

**Comment by hongkailiu** [comment]
We have got the green light from TRT.

Feel free to cancel the hold whenever you are ready.

**Comment by haircommander** [comment]
/hold cancel

thanks!

**Pull request merged**